src/actions/userManagement.js

```diff
@@ -126,3 +132,29 @@ export const userProfileDeleteAction = (user) => {
     user,
   };
 };
+/**
+ * update the user profile
+ * @param {*} user - the user to be updated
+ * @param {*} finalDate  - the date to be inactive
+ */
+export const updateUserFinalDayStatus = (user,status,finalDayDate) => {
```

Comment:
Pls add space trailing the commas, also add spaces around line 145 parentheses and around line 148 `else`, according to [Google JS style guide](https://google.github.io/styleguide/jsguide.html#formatting-braces)

Reply:
Thx, already added space after commas, around parentheses, and reserved keywords.